tests/Rules/Data/model-property-builder.php

```diff
@@ -22,6 +22,9 @@ function bar(\Illuminate\Database\Eloquent\Builder $builder): \Illuminate\Databa
 
 \App\User::query()->whereColumn('foo', '=', 'foo');
 \App\User::query()->where(getKey(), '=', 'foo');
+\App\User::query()->orWhere('foo', '=', 'foo');
+\App\User::query()->orWhere('foo', 'foo');
```

Comment:
`orWhere` also accepts arrays. With the array key being the model property. So can you add a test case like
```php
\App\User::query()->orWhere([
	'foo' => 'bar'
]);
```